packages/next/src/shared/lib/normalized-asset-prefix.ts

```diff
@@ -1,11 +1,11 @@
 export function normalizedAssetPrefix(assetPrefix: string | undefined): string {
-  const escapedAssetPrefix = assetPrefix?.replace(/^\/+/, '') || false
-
-  // assetPrefix as a url
-  if (escapedAssetPrefix && escapedAssetPrefix.startsWith('://')) {
-    return escapedAssetPrefix.split('://', 2)[1]
+  // if assetPrefix is a URL, we return the pathname
```

Comment:
This comment is just repeating what the code says. The "what" should only be explained if the code is not trivial which it should be 99% of the time. The "why" is more important and that's missing here.

Reply:
Will keep this in my heart, thank you.